installation/kie-helm-charts.adoc

```diff
@@ -0,0 +1,409 @@
+
+[source, shell]
+----
+NAME: kie-sandbox
+LAST DEPLOYED: Wed Nov 29 17:09:04 2023
+NAMESPACE: default
+STATUS: deployed
+REVISION: 1
+NOTES:
+1. CORS Proxy available at:
+  http://cors-proxy.<YOUR_KUBERNETES_CLUSTER_DOMAIN>
+2. Extended Services available at:
+  http://extended-services.<YOUR_KUBERNETES_CLUSTER_DOMAIN>
+3. KIE Sandbox available at:
+  http://kie-sandbox.<YOUR_KUBERNETES_CLUSTER_DOMAIN>
+----
+
+You need not run any commands. The {CANVAS}
+ can be accessible via https://kie-sandbox.<YOUR_KUBERNETES_CLUSTER_DOMAIN>
+
+=== OpenShift install
+
+First, you may need to get the default OpenShift domain for your routes with the following command:
+
+[source, shell]
+----
+$ oc get ingresses.config cluster --output jsonpath={.spec.domain}
+----
+
+If you do not have access rigths to this configutation, try creating a dummy Route resource and check its domain.
+
+Follow the command to install the chart with the release name `kie-sandbox`:
+
+[source, shell]
+----
+$ helm install kie-sandbox ./src --values ./src/values-openshift.yaml --set global.openshiftRouteDomain="<YOUR_OCP_ROUTE_DOMAIN>"
+----
+
+Following message must be displayed on your console.
+
+[source, shell]
+----
+NAME: kie-sandbox
+LAST DEPLOYED: Wed Nov 29 17:09:04 2023
+NAMESPACE: default
+STATUS: deployed
+REVISION: 1
+NOTES:
+1. CORS Proxy available at:
+  https://cors-proxy.<YOUR_OCP_ROUTE_DOMAIN>
+2. Extended Services available at:
+  https://extended-services.<YOUR_OCP_ROUTE_DOMAIN>
+3. KIE Sandbox available at:
+  https://kie-sandbox.<YOUR_OCP_ROUTE_DOMAIN>
+----
+
+You need not run any commands. The BAMOE CANVAS must be accessible via https://kie-sandbox.<YOUR_OCP_ROUTE_DOMAIN>
+
+== Installing a released version from the OCI registry:
+
+It is very similar to the way you install the chart from source code. You can also install a released version available on quay.io registry:
+
+=== Default install
+
+[source, shell]
+----
+$ helm install kie-sandbox oci://quay.io/kie-tools/kie-sandbox-helm-chart --version=0.0.0
+----
+
+=== Minikube install
+
+[source, shell]
+----
+$ helm pull oci://quay.io/kie-tools/kie-sandbox-helm-chart --version=0.0.0 --untar
+$ helm install kie-sandbox ./kie-sandbox-helm-chart --values ./kie-sandbox-helm-chart/values-minikube-nginx.yaml
+----
+
+=== Kubernetes install
+
+[source, shell]
+----
+$ helm pull oci://quay.io/kie-tools/kie-sandbox-helm-chart --version=0.0.0 --untar
+$ helm install kie-sandbox ./kie-sandbox-helm-chart --values ./kie-sandbox-helm-chart/values-kubernetes.yaml --set global.kubernetesClusterDomain="<YOUR_KUBERNETES_CLUSTER_DOMAIN>" --set global.kubernetesIngressClass="<YOUR_KUBERNETES_INGRESS_CLASS>"
+----
+
+=== OpenShift install
+
+[source, shell]
+----
+$ helm pull oci://quay.io/kie-tools/kie-sandbox-helm-chart --version=0.0.0 --untar
+$ helm install kie-sandbox ./kie-sandbox-helm-chart --values ./kie-sandbox-helm-chart/values-openshift.yaml --set global.openshiftRouteDomain="<YOUR_OCP_ROUTE_DOMAIN>"
```

Comment:
Here we should also mention how to get the OCP_ROUTE_DOMAIN variable (just copy from the OpenShift install steps above)

Reply:
@thiagoelg  - I'm not able to find the steps to get the OCP_ROUTE_DOMAIN variable in the OpenShift install section. Can you please help?

Reply:
It's this part:

```

First, you may need to get the default OpenShift domain for your routes with the following command:

[source, shell]
----
$ oc get ingresses.config cluster --output jsonpath={.spec.domain}
----

If you do not have access rigths to this configutation, try creating a dummy Route resource and check its domain.
```